packages/frontend-app/src/app/components/Card/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+import "./index.scss";
+
+interface IProps {
+  src: string;
+  title: string;
+  artist: string;
+}
+
+export const Card: React.FC<IProps> = ({
+  src,
+  title = "I told them...",
+  artist = "Burna Boy",
+}) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleMouseOver = () => {
+    setIsHovered(true);
+  };
+
+  const handleMouseOut = () => {
+    setIsHovered(false);
+  };
+
+  return (
+    <div
+      className="flex flex-col cursor-pointer w-[214px]"
+      onMouseOver={handleMouseOver}
+      onMouseOut={handleMouseOut}
+    >
+      <div className="relative">
+        <img
+          src={src || "public/images/burna.jpeg"}
+          alt={title}
+          width={214}
+          height={214}
+        />
+        {isHovered && (
+          <div className="overlay">
+            <div className="overlay-content">
+              <button className="text-white text-sm">play</button>
+              <button className="text-white text-sm">options</button>
```

Comment:
We have reusable buttons you can use 😅 . Check the `Button` component. You can use the variant `contained` for now